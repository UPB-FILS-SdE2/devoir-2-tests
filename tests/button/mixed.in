button a
button b
quit