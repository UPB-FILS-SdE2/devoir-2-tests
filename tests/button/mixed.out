cmd: True
cmd: False
cmd: 